demo/app/src/main/java/org/pf4j/demo/Boot.java

```diff
@@ -56,53 +56,36 @@ public static void main(String[] args) {
 
         // print extensions from classpath (non plugin)
         System.out.println("Extensions added by classpath:");
-        Set<String> extensionClassNames = pluginManager.getExtensionClassNames(null);
-        for (String extension : extensionClassNames) {
-            System.out.println("   " + extension);
-        }
+        printIndented(pluginManager.getExtensionClassNames(null));
 
         System.out.println("Extension classes by classpath:");
-        List<Class<Greeting>> greetingsClasses = pluginManager.getExtensionClasses(Greeting.class);
-        for (Class<Greeting> greeting : greetingsClasses) {
+        for (Class<Greeting> greeting : pluginManager.getExtensionClasses(Greeting.class)) {
             System.out.println("   Class: " + greeting.getCanonicalName());
```

Comment:
why not `printIndented` here

Reply:
I changed it to use printIndented, the code is longer due to this change.
On the other hand, we prevented a bug in case someone changes the indentation style
only in `printIndented` and not in `System.out.println("   Class: " + greeting.getCanonical...`